ui/redux/actions/user.js

```diff
@@ -784,3 +831,30 @@ export function doCheckYoutubeTransfer() {
       });
   };
 }
+
+export function doFetchUserMemberships(claimIdCsv: string) {
+  return (dispatch) => {
+    (async function () {
+      const response = await Lbryio.call('membership', 'check', {
+        channel_id: '80d2590ad04e36fb1d077a9b9e3a8bba76defdf8',
+        claim_ids: claimIdCsv,
+      });
+
+      let updatedResponse = {};
+
+      for (const user in response) {
+        if (response[user] && response[user].length) {
+          for (const membership of response[user]) {
+            if (membership.channel_name) {
+              updatedResponse[user] = membership.name;
+            }
+          }
+        } else {
+          updatedResponse[user] = null;
+        }
+      }
+
+      dispatch({ type: ACTIONS.ADD_CLAIMIDS_MEMBERSHIP_DATA, data: { response: updatedResponse } });
+    })();
```

Comment:
Creating an async and running it feels weird.  Would changing `return (dispatch)...` to `return async (dispatch)...` achieved the same thing, while being conformant and readable?